src/app/server/Dnssd.cpp

```diff
@@ -420,6 +420,11 @@ void DnssdServer::StartServer(chip::Dnssd::CommissioningMode mode)
     {
         ChipLogError(Discovery, "Failed to initialize advertiser: %s", chip::ErrorStr(err));
     }
+    err = Dnssd::Resolver::Instance().Init(&chip::DeviceLayer::InetLayer());
```

Comment:
With this change the resolver will be initialized by default for everyone calling `StartServer`. Is this expected?